templates/etc/php/7.4

```diff
@@ -0,0 +1 @@
+7.0
```

Comment:
I don't get this, does it just point the 7.4 install to use the `templates/etc/php/7.0/fpm/pool.d/atom.conf`?

Reply:
Yes - just a symlink. The PHP 7.2 version is doing the same thing.